### PR TITLE
warm up network for test read unapplied leader

### PR DESCRIPTION
**Pull request by zhangjinpeng1987**
warm up network for test read unapplied leader

### CODE REVIEW

File: tests/raftstore/test_multi.rs [hunk 1]
```diff
@@ -371,6 +371,16 @@ fn test_read_leader_with_unapplied_log<T: Simulator>(cluster: &mut Cluster<T>) {
     // guarantee peer 1 is leader
     cluster.must_transfer_leader(1, new_peer(1, 1));
 
+    // if 2 is unreachable, leader will not send MsgAppend to 2, and will send MsgAppend
+    // with committed information to 2 later, and 2 will apply the entry regardless of we
+    // add an filter.
+    let (k_warm, v_warm) = (b"k_warm", b"v_warm");
+    cluster.must_put(k_warm, v_warm);
+
+    must_get_equal(&cluster.get_engine(1), k_warm, v_warm);
```
Comment (siddontang) [inline tests/raftstore/test_multi.rs]: you can use:

```
for i in 1..4 {
must_get_equal( cluster.get_engine(i), k_warn, v_warm)
}
```

File: tests/raftstore/test_multi.rs [hunk 2]
```diff
@@ -371,6 +371,16 @@ fn test_read_leader_with_unapplied_log<T: Simulator>(cluster: &mut Cluster<T>) {
     // guarantee peer 1 is leader
     cluster.must_transfer_leader(1, new_peer(1, 1));
 
+    // if 2 is unreachable, leader will not send MsgAppend to 2, and will send MsgAppend
```
Comment (ngaut) [inline tests/raftstore/test_multi.rs]: s/2/peer 2/

**Comment by siddontang** [comment]
LGTM

File: tests/raftstore/test_multi.rs [hunk 3]
```diff
@@ -371,6 +371,16 @@ fn test_read_leader_with_unapplied_log<T: Simulator>(cluster: &mut Cluster<T>) {
     // guarantee peer 1 is leader
     cluster.must_transfer_leader(1, new_peer(1, 1));
 
+    // if peer 2 is unreachable, leader will not send MsgAppend to peer 2, and will send MsgAppend
```
Comment (ngaut) [inline tests/raftstore/test_multi.rs]: The comment makes me confused.

Comment (ngaut) [inline tests/raftstore/test_multi.rs]: what does key k_warm mean ?

Comment (zhangjinpeng1987) [inline tests/raftstore/test_multi.rs]: No special meaning

Comment (ngaut) [inline tests/raftstore/test_multi.rs]: Then you should use normal variable name and value.

**Comment by ngaut** [comment]
LGTM

**Pull request merged**